sources/procfs.go

```diff
@@ -125,68 +125,68 @@ func newLustreProcMetric(filename string, promName string, source string, path s
-			{"job_stats", "job_write_samples", writeSamplesHelp},
-			{"job_stats", "job_write_minimum", writeMinimumHelp},
-			{"job_stats", "job_write_maximum", writeMaximumHelp},
-			{"job_stats", "job_write_total", writeTotalHelp},
-			{"job_stats", "job_num_getattr", getattrHelp},
-			{"job_stats", "job_num_setattr", setattrHelp},
-			{"job_stats", "job_num_punch", punchHelp},
-			{"job_stats", "job_num_sync", syncHelp},
-			{"job_stats", "job_num_destroy", destroyHelp},
-			{"job_stats", "job_num_create", createHelp},
-			{"job_stats", "job_num_statfs", statfsHelp},
-			{"job_stats", "job_num_get_info", getInfoHelp},
-			{"job_stats", "job_num_set_info", setInfoHelp},
-			{"job_stats", "job_num_quotactl", quotactlHelp},
-			{"kbytesavail", "kbytesavail", "Number of kilobytes readily available in the pool"},
-			{"kbytesfree", "kbytesfree", "Number of kilobytes allocated to the pool"},
-			{"kbytestotal", "kbytestotal", "Capacity of the pool in kilobytes"},
-			{"lfsck_speed_limit", "lfsck_speed_limit", "Maximum operations per second LFSCK (Lustre filesystem verification) can run"},
-			{"num_exports", "num_exports", "Total number of times the pool has been exported"},
-			{"precreate_batch", "precreate_batch", "Maximum number of objects that can be included in a single transaction"},
-			{"recovery_time_hard", "recovery_time_hard", "Maximum timeout 'recover_time_soft' can increment to for a single server"},
-			{"recovery_time_soft", "recovery_time_soft", "Duration in seconds for a client to attempt to reconnect after a crash (automatically incremented if servers are still in an error state)"},
-			{"soft_sync_limit", "soft_sync_limit", "Number of RPCs necessary before triggering a sync"},
-			{"stats", "read_samples_total", readSamplesHelp},
-			{"stats", "read_minimum_size_bytes", readMinimumHelp},
-			{"stats", "read_maximum_size_bytes", readMaximumHelp},
-			{"stats", "read_total_bytes", readTotalHelp},
-			{"stats", "write_samples_total", writeSamplesHelp},
-			{"stats", "write_minimum_size_bytes", writeMinimumHelp},
-			{"stats", "write_maximum_size_bytes", writeMaximumHelp},
-			{"stats", "write_total_bytes", writeTotalHelp},
-			{"sync_journal", "sync_journal", "Binary indicator as to whether or not the journal is set for asynchronous commits"},
-			{"tot_dirty", "tot_dirty", "Total number of exports that have been marked dirty"},
-			{"tot_granted", "tot_granted", "Total number of exports that have been marked granted"},
-			{"tot_pending", "tot_pending", "Total number of exports that have been marked pending"},
+			{"blocksize", "ost_blocksize_bytes", "Filesystem block size in bytes"},
+			{"brw_size", "ost_brw_size_total", "Block read/write size in bytes"},
+			{"brw_stats", "ost_pages_per_bulk_rw_total", pagesPerBlockRWHelp},
+			{"brw_stats", "ost_discontiguous_pages_total", discontiguousPagesHelp},
+			{"brw_stats", "ost_disk_io_now", diskIOsInFlightHelp},
+			{"brw_stats", "ost_io_time_ms", ioTimeHelp},
```

Comment:
`IO Time` is a counter, so probably want to change to `ost_io_time_ms_total`, yes?